examples/project-service-list-simple/README.md

```diff
@@ -0,0 +1,33 @@
+# project-service-list: Simple Example
+
+### Overview
+
+In this example, we will see how to generate [project services](https://cloud.google.com/config-connector/docs/reference/resource-docs/serviceusage/service) for `compute.googleapis.com` and `redis.googleapis.com`.
+
+### Fetch the example package
+
+Get the example package by running the following commands:
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/project-service-list-simple
+```
+
+### Function invocation
+
+Render the configuration, `project-service-list` has been defined declaratively in the Kptfile and configured via `services.yaml`.
+
+```shell
+$ kpt fn render project-service-list-simple --results-dir /tmp
```

Comment:
since we are specifying `results-dir` here, we should show how to examine the results. Or we can remove `results-dir` ?

Reply:
I think we can remove `results-dir` here. Most use these results in this case comes from stdout.

Reply:
removed